common/changes/@itwin/editor-frontend/fix-builtin-edit-commands_2022-09-01-16-21.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/editor-frontend",
+      "comment": "Fix spelling in method docs.",
```

Comment:
```suggestion
      "comment": "",
```

Reply:
Why is this important? If we aren't supposed to describe the change for "rush change" make it not ask me?

Reply:
This is just a nit; feel free to ignore.

The rush change is used to generate the changelog. imo i dont think we should care about spelling changes or anything non api related that we want to explicitly call out

Reply:
Okay, I will keep that rule in mind for the future,